src/main/java/Parser.java

```diff
@@ -0,0 +1,56 @@
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class Parser {
+    public boolean scan() {
+        Scanner sc = new Scanner(System.in);
+        boolean isLoop = true;
+
+        try {
+            switch (sc.next().toLowerCase()) {
```

Comment:
I like the use of a switch statement here which makes the code readable and easy to follow!